sechub-scan/src/main/java/com/mercedesbenz/sechub/domain/scan/ScanTypeCount.java

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.domain.scan;
+
+import com.mercedesbenz.sechub.commons.model.ScanType;
+
+public class ScanTypeCount {
```

Comment:
Please make the class comparable based on the ScanType's natural ordering (B A C -> becomes -> A B C).

Reply:
Done